profile/README.md

```diff
@@ -41,17 +51,33 @@ pyenv local sense_EEG-env
 pip install -e ".[dev]"
```

Comment:
I'm not familiar with the [dev] syntax - what does this do?

Reply:
No prob. 

`.[dev]` will install developer dependencies stated in pyproject.toml. This is useful to install just what you need. See an example for sense_eeg; https://github.com/sense-base/sense_eeg/blob/main/pyproject.toml#L8